### DISCUSSION TITLE
Single SID Emulation with One 74CBTD3861 on SIDKick Pico?

### DISCUSSION THREAD

**Discussion by serkantanriverdi** [frntc/SIDKick-pico#79] [Q&A]
If I'm not mistaken, the purpose of the two 74CBTD3861 chips on the SIDKick Pico is to enable dual SID emulation. I'm curious, can we run single SID emulation by mounting only one 74CBTD3861 chip, or are two chips required in all cases? If a single 74CBTD3861 is sufficient, which one on the PCB is responsible for single SID emulation? Thank you.

**Comment by frntc** [comment]
You also need both level shifters/74CBTD3861 -- there are two many signals involved (the address lines for the 2nd SID only add two signals).
